src/pages/crop.tsx

```diff
@@ -0,0 +1,598 @@
+import { NextPage } from "next";
+import { Layout } from "@/components/Layout";
+import Image from "next/image";
+import { useStores } from "@/stores/context";
+import React, { useEffect, useState } from "react";
+import { useAuthState } from "react-firebase-hooks/auth";
+import { auth } from "@/service/firebase";
+import router from "next/router";
+import cropStyles from "@/styles/crop.module.scss";
+import cropListPopupStyles from "@/styles/cropListPopup.module.scss";
+import { observer } from "mobx-react";
+import { Crops, CROPS } from "@/constants/crops";
+import { GrowingCrop } from "@/models/crop/GrowingCrop";
+import { fruit_grade } from "@prisma/client";
+import Modal from "react-modal";
+import emptyImage from "@/assets/Frame 586.png";
+
+const crop: NextPage = () => {
+  return (
+    <Layout>
+      <h1 className={"typography__sub-headline"}>내 작물 관리하기</h1>
+      <MyPot></MyPot>
+      <MyPlants></MyPlants>
+      <style jsx>{`
+        h1 {
+          margin-left: 20px;
+        }
+      `}</style>
+    </Layout>
+  );
+};
+
+const MyPot: NextPage = observer(() => {
+  const { cropStore } = useStores();
+  const [user, loading] = useAuthState(auth);
+  const [exist, setExist] = useState<boolean>(false);
+  const [open, setOpen] = useState(false);
+
+  if (loading) {
+    return <div>Loading</div>;
+  }
+  if (!user) {
+    router.replace("/login");
+    return <div>Please sign in to continue</div>;
+  }
+
+  useEffect(() => {
+    cropStore.fetchGrowingCrop(user.uid);
+    cropStore.fetchHarvestedCrops(user.uid);
+  }, []);
+
+  useEffect(() => {
+    if (cropStore.growingCrop != null && !cropStore.growingCrop.isDead) {
+      setExist(true);
+    }
+  }, [cropStore.growingCrop]);
+
+  return (
+    <div className={`${cropStyles["my-pot"]}`}>
+      {cropStore.growingCrop != undefined &&
```

Comment:
```suggestion
      {cropStore.growingCrop !== undefined &&
```
이렇게 하는게 더 안전하지 않을까요 ?!

Reply:
null 이랑 undefined 둘다 체크하려고 이렇게 했습니다.